awesomerc.lua

```diff
@@ -506,7 +506,8 @@ awful.rules.rules = {
           "Sxiv",
           "Wpa_gui",
           "veromix",
-          "xtightvncviewer"},
+          "xtightvncviewer",
+          "Tor Browser"},
```

Comment:
Just noticed that the entries are sorted, so please adjust it - a good exercise maybe for your first PR.
Please amend the existing commit and force-push it then.
But just let us know if this is too much, and we can take it from here also.